packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -57,11 +57,15 @@ const QuickStrategyForm = ({
             const validation_errors = validateQuickStrategy(values);
             const is_valid = Object.keys(validation_errors).length === 0;
             const is_submit_enabled = !isSubmitting && is_valid;
-            const form_margin = is_mobile ? 'calc(100% - 72px)' : '430px';
+            const form_margin = '430px';
```

Comment:
Since it's just used in one place, could you omit this`const form_margin = '430px';` and use it directly.